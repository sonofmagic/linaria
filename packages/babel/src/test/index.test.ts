import fs from 'fs';
import path from 'path';

import { transformAsync } from '@babel/core';

import linaria from '../index';

describe('default', () => {
  it('default', async () => {
    const code = fs.readFileSync(
      path.resolve(__dirname, './fixtures', '0.default.jsx'),
      'utf8'
    );
    const res = await transformAsync(code, {
      filename: __filename,
      presets: ['@babel/preset-env', '@babel/preset-react', linaria],
    });
    expect(res?.code).toMatchSnapshot();
  });
});
