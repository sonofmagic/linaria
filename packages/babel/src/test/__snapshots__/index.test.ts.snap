// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default default 1`] = `
"\\"use strict\\";

var header = \\"h1pelf21\\";"
`;
